spring-boot/admin/src/main/resources/application.yml

```diff
@@ -75,6 +75,31 @@ spring:
     rest:
       base-path: /api
 
+
+  security:
+    oauth2:
+      client:
+        registration:
+         # resource:
+          #  userInfoUri: https://api.github.com/user
```

Comment:
why not delete these two lines?

Reply:
This might be important for the coatrack#183, so I added the TODO